mod/wet4/views/default/page/layouts/elements/header.php

```diff
@@ -48,6 +48,13 @@
             'class' => 'list-inline mrgn-rght-sm',
             ));
         }
+        if($checkPage == 'messages') {
+            $notificationSettings = elgg_echo('cp_notifications:name');
+
+            $notificationsSettingLink = "{$_SERVER['SERVER_NAME']}/settings/notifications/";
```

Comment:
elgg has a built-in function for the root url, elgg_get_site_url()
also if you wanted to add the username here elgg_get_logged_in_user_entity() will get you that, and that object has a ->username property, as in
```php
$user_object = elgg_get_logged_in_user_entity();
$username = $user_object->username
```

in this case though it seems to work without the username but this will probably be useful later

Reply:
Oh thanks for that tip, didn't know about that built in function. I've updating the PR with the changes you suggested. I will reach out to Nick and see what he thinks about where I but the link.